### ISSUE TITLE
Deleting a media library image that was attached to a Modula gallery throws a fatal error

### ISSUE THREAD

**Issue by cristianraiber** [WPChill/modula-lite#910]
Fatal error: Uncaught Error: Cannot use object of type WP_Error as array in .........../public_html/wp-content/plugins/modula-best-grid-gallery/includes/public/modula-helper-functions.php:57 Stack trace:

It seems like this line $resized = $resizer->resize_image( $sizes['url'], $sizes['width'], $sizes['height'], $crop ); is generating a Fatal Error

Modula Plugins Active: Modula Version 2.7.91 by WPChill | Auto-updates disabled Modula - PRO Version 2.6.9 by WPChill | Auto-updates disabled Modula Slideshow Version 1.0.6 by WPChill | Auto-updates disabled Modula Video Version 1.0.11 by WPChill | Auto-updates disabled

The error is thrown on line 57 here
/wp-content/plugins/modula-best-grid-gallery/includes/public/modula-helper-functions.php:57

if on the blank line above (56) I capture the error output if ( !is_array($sizes)  I get this:

WP_Error Object
(
    [errors] => Array
        (
            [modula-gallery-error-no-url] => Array
                (
                    [0] => No image with this ID.
                )

        )

    [error_data] => Array
        (
        )

    [additional_data:protected] => Array
        (
        )

)

**Comment by cristianraiber** [comment]
```The error is thrown on line 57 here
/wp-content/plugins/modula-best-grid-gallery/includes/public/modula-helper-functions.php:57

if on the blank line above (56) I capture the error output if ( !is_array($sizes)  I get this:

WP_Error Object
(
    [errors] => Array
        (
            [modula-gallery-error-no-url] => Array
                (
                    [0] => No image with this ID.
                )

        )

    [error_data] => Array
        (
        )

    [additional_data:protected] => Array
        (
        )

)
```

**Comment by cristeacosmin** [comment]
should be fixed by #947
